tests/support/server.tcl

```diff
@@ -585,7 +586,7 @@ proc start_server {options {code undefined}} {
 
         while 1 {
             # check that the server actually started and is ready for connections
-            if {[count_message_lines $stdout "Ready to accept"] > 0} {
+            if {[count_message_lines $stdout "Ready to accept"] == [expr $before_start_ready_count + 1]} {
```

Comment:
Just like `if {[count_message_lines $stdout "Ready to accept"] > $prev_ready_count}` at the bottom.
```suggestion
            if {[count_message_lines $stdout "Ready to accept"] > expr $before_start_ready_count} {
```